### PR TITLE
SEOImages: User Story 1463880, Part 19

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Files #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report:
https://contentqacats.azurewebsites.net/CheckResults/2019031700011/Report/

There is one image name identified by CATS as being too short. It should be handled at a later time as it is not part of the SEO image without hyphen work.

File: docs/framework/wpf/advanced/how-to-use-automatic-layout-to-create-a-button.md [hunk 1]
```diff
@@ -20,8 +20,7 @@ This example describes how to use the automatic layout approach to create a butt
   
  The following graphic shows the output of the code samples.
```
Comment (mairaw) [inline docs/framework/wpf/advanced/how-to-use-automatic-layout-to-create-a-button.md]:
```suggestion
 The following graphic shows the output of the code samples with auto-resizable buttons:
```

File: docs/visual-basic/programming-guide/concepts/serialization/index.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Serialization is the process of converting an object into a stream of bytes in o
 ## How Serialization Works  
  This illustration shows the overall process of serialization.  
   
- ![Serialization Graphic](../../../../csharp/programming-guide/concepts/serialization/media/serialization.gif "serialization")  
+![Serialization graphic](./media/index/serialization-graphic.gif)
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/concepts/serialization/index.md]:
```suggestion
![Serialization graphic](./media/index/serialization-process.gif)
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you @damabe. A few comments before we merge.

**Comment by damabe** [comment]
@mairaw Thanks for the feedback! Changes are made. New CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031900004/Report/

There is one image name identified by CATS as being too short. It should be handled at a later time as it is not part of the SEO image without hyphen work.

**Review by mairaw — APPROVED** [review]
LGTM ✌️

**Pull request merged**